d3a_api_client/utils.py

```diff
@@ -60,6 +60,7 @@ def execute_graphql_request(domain_name, query, headers=None, url=None, authenti
     if authenticate:
         jwt_key = retrieve_jwt_key_from_server(domain_name)
         if jwt_key is None:
+            logging.error(f"authentication failed")
```

Comment:
Minor, but I would not leave this in "error" level because it could potentially spam our logs.

Reply:
@hannesdiedrich Isn't this logged only on the client side? Shouldn't the program fail if the authentication is not successful?

Reply:
@fievelk You are right, this function is used both for the normal client and the WSComsumer which we will run on our cluster. So, leave it as is for now. If we get spammed by this error message, we can deal with it later.